### PR TITLE
Check that value != 0 before taking the log

### PR DESCRIPTION
**Pull request by guiniol**
`log(0)` is not something we want to do, so check that value != 0 before doing any such thing.

### CODE REVIEW

File: py3status/py3.py [hunk 1]
```diff
@@ -235,7 +235,7 @@ def format_units(self, value, unit='B', optimal=5, auto=True, si=False):
                 value /= size
                 can_round = True
 
-        if can_round and optimal:
+        if can_round and optimal and not value == 0:
```
Comment (tobes) [inline py3status/py3.py]: good catch

`if can_round and optimal and value:` feels more readable

Comment (guiniol) [inline py3status/py3.py]: Well, my code caught it ^^

**Comment by guiniol** [comment]
Also, use the `abs` in the log, in case the value is negative.

**Comment by tobes** [comment]
Ok looks good I'll add some tests as I have a branch for that including the 0 case and better errors but it needs these fixes

**Comment by ultrabug** [comment]
Thanks !

**Pull request merged**